src/settingslist.h

```diff
@@ -783,3 +785,23 @@ SETTING_BOOL(addonApiSetting,        // getter
              false                   // sensitive (do not log)
 )
 #endif
+
+SETTING_UINT(extensionTelemetryFlags,        // getter
```

Comment:
This feels like an anti-pattern for Glean, and has some possible data retention concerns. The point of Glean is that it handles recording our metrics, and the lifetimes should be able to handle our use cases.

Reply:
You are totally right, TIL about `lifetime =! ping` - ripped all of that code out and moved that into the metrics.yaml